src/libs/ajax/ajax-common.test.ts

```diff
@@ -108,48 +99,77 @@ describe('withRetryAfterReloadingExpiredAuthToken', () => {
       expect(loadAuthToken).toHaveBeenCalled();
     });
 
-    it('retries request with new auth token if reloading auth token succeeds', async () => {
+    describe('and reloading auth token fails', () => {
```

Comment:
Nit: I think these read a little clearer with "if" instead of "and".

```suggestion
    describe('if reloading auth token fails', () => {
```